extension/lib/BiasDoorhanger.jsm

```diff
@@ -137,7 +137,9 @@ class BiasDoorhanger {
       const timeSinceShown = Date.now() - stats.timestamp;
       const shouldShow = !stats.neverAgain && timeSinceShown > Config.showDoorhangerInterval;
 
-      if (hostname && isTreatmentPhase && isTracked && shouldShow) {
+      const inTreatmentBranch = Pioneer.utils.chooseBranch().showDoorhanger;
```

Comment:
How does this interact with overriding the branch via a pref? Does pioneer-utils need to read the pref/read from config to read the override pref? Otherwise we should probably store the branch in memory somewhere.

There is also the concern that this is a moderately expensive operation, since it does some hashing. But pioneer-utils can easily cache it to avoid that.

Finally, `chooseBranch()` is (or should be, at least) async, so this needs to await it.